barcelona.html

```diff
@@ -533,7 +536,7 @@ <h3>Follow us</h3>
 
       <div class="flex flex-col gap-5 text-lg lg:text-xl text-slate-100">
         <h3>Get in touch</h3>
-        <p><a href="mailto:hello@codebar.io">hello@codebar.io</a></p>
+        <p><a href="mailto:hello@codebar.io">barcelona@codebar.io</a></p>
```

Comment:
oh the hello@codebar.io needs to change here too.

Reply:
fixed :)